### ISSUE TITLE
[Bug]: milvus fail to post object to cfs object storage

### ISSUE THREAD

**Issue by Ghostbaby** [cubefs/cubefs#2389]
**Describe the bug**
milvus error message:
```
[2023/08/02 02:10:24.213 +00:00] [WARN] [storage/minio_chunk_manager.go:203] ["failed to put object"] [path=insert_log/443251913744189007/443251913744189008/443251913744389682/101/443251913744954792] [error="Post \"http://***********:17410/milvus/insert_log/443251913744189007/443251913744189008/443251913744389682/101/443251913744954792?uploadId=\": Connection closed by foreign host http://***********:17410/milvus/insert_log/443251913744189007/443251913744189008/443251913744389682/101/443251913744954792?uploadId=. Retry again."]
```

cubefs objectNode warn message:
```
2023/08/02 10:47:03.981944 [WARN ] fs_volume.go:1124: AbortMultipart: unlink part inode: volume(milvus) path(insert_log/443251913744189007/443251913744189008/443251913744389682/101/443251913744954792) multipartID(3244eca8e3ae439196eba107269e38cex014f957617b762f490aa3158) partID(1) inode(33566415)
2023/08/02 10:47:03.983113 [WARN ] fs_volume.go:1130: AbortMultipart: evict part inode: volume(milvus) path(insert_log/443251913744189007/443251913744189008/443251913744389682/101/443251913744954792) multipartID(3244eca8e3ae439196eba107269e38cex014f957617b762f490aa3158) partID(1) inode(33566415)
2023/08/02 10:47:03.983816 [WARN ] fs_volume.go:1124: AbortMultipart: unlink part inode: volume(milvus) path(insert_log/443251913744189007/443251913744189008/443251913744389682/101/443251913744954792) multipartID(3244eca8e3ae439196eba107269e38cex014f957617b762f490aa3158) partID(2) inode(16789184)
2023/08/02 10:47:03.985079 [WARN ] fs_volume.go:1130: AbortMultipart: evict part inode: volume(milvus) path(insert_log/443251913744189007/443251913744189008/443251913744389682/101/443251913744954792) multipartID(3244eca8e3ae439196eba107269e38cex014f957617b762f490aa3158) partID(2) inode(16789184)
```

**Environment**
OS:
NAME="CentOS Linux"
VERSION="8"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="CentOS Linux 8"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:8"
HOME_URL="https://centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"
CENTOS_MANTISBT_PROJECT="CentOS-8"
CENTOS_MANTISBT_PROJECT_VERSION="8"

CubeFS Server:
Version : v3.2.1
Branch  : heads/v3.2.1
Commit  : 00e7ae59d56508e1926862adf83ef4329819ae6a
Build   : go1.17.12 linux amd64 2023-04-23 19:18

**To Reproduce**

**Expected behavior**

**Additional context**

**Comment by tangdeyi** [comment]
The bug has been fixed in v3.3.0 which will be released soon.

**Comment by Ghostbaby** [comment]
@tangdeyi update cubefs version to develop-3.3.0-stable branch, this issue fixed.
